static_test_driver/sensors.ino

```diff
@@ -1,3 +1,9 @@
+float mean (const float *data, unsigned int size) {
+  float result;
+  for (int i = 0; i < size; i++)
+      result += data[i];
+  return result /= size;
```

Comment:
Quick style thing: No need to assign to result, you can just `return result / size;`